index.js

```diff
@@ -190,6 +190,23 @@ class HyperdriveFuse {
       })
     }
 
+    handlers.rename = function (oldpath, newpath, cb) {
+      log('rename', oldpath, newpath)
+      // This implementation only supports file rename, directories isn't really feasible for the moment
+      // TODO: Provide better renaming support once new trie rolls out
+      self.drive.stat(oldpath, (err,stat) =>{
+        if(stat.isFile()){
+          self.drive.copy(oldpath,newpath, (err, stat) => {
+          if (err) return cb(-err.errno || Fuse.ENOENT)
+            self.drive.unlink(oldpath, err => {
+              if (err) return cb(-err.errno || Fuse.ENOENT)
+              return cb(0)
+            })
+          })
+        }
+      })
```

Comment:
Also need to have an `else` clause for the directory case that throws an EISDIR, but based on our Discord chat I'm guessing you were gonna add that :)